nntrainer/app_context.h

```diff
@@ -262,7 +278,7 @@ class AppContext {
   }
 
 private:
-  FactoryMap<ml::train::Optimizer, ml::train::Layer> factory_map;
+  FactoryMap<ml::train::Optimizer, nntrainer::Layer> factory_map;
```

Comment:
ml::train::Optimizer --> nntrainer::Optimizer later?

Reply:
This can be done later, as it does not provide any advantage.